ksql-rest-app/src/test/resources/ksql-plan-schema/schema-final.json

```diff
@@ -0,0 +1,885 @@
+    },
+    "StreamGroupByKey" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamGroupByKeyV1" ],
+          "default" : "streamGroupByKeyV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        }
+      },
+      "title" : "streamGroupByKeyV1",
+      "required" : [ "@type", "properties", "source", "formats" ]
+    },
+    "StreamMapValues" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamMapValuesV1" ],
+          "default" : "streamMapValuesV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "selectExpressions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "selectNodeName" : {
+          "type" : "string"
+        }
+      },
+      "title" : "streamMapValuesV1",
+      "required" : [ "@type", "properties", "source", "selectExpressions", "selectNodeName" ]
+    },
+    "StreamSelectKey" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamSelectKeyV1" ],
+          "default" : "streamSelectKeyV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "fieldName" : {
+          "type" : "string"
+        },
+        "updateRowKey" : {
+          "type" : "boolean"
+        }
+      },
+      "title" : "streamSelectKeyV1",
+      "required" : [ "@type", "properties", "source", "fieldName", "updateRowKey" ]
+    },
+    "StreamSink" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamSinkV1" ],
+          "default" : "streamSinkV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        },
+        "topicName" : {
+          "type" : "string"
+        }
```

Comment:
We've `kafakTopicName` in our commands, but `topicName` here.  We should pick on and stick to it.  Personally, I'd go with `topicName`.